pipelines/pipelines/cnbi-package-list.yaml

```diff
@@ -0,0 +1,204 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: cnbi-package-list
+  annotations:
+    meteor.zone/pipeline: "true"
+    shower.meteor.zone/button-label: Run code in JupyterHub
+spec:
+  description: Build a notebook image from a package list
+  params:
+    - name: baseImage
+      description: Container image repository url
+      type: string
+    - name: packages
+      type: array
+    - name: name
+      description: Image name
+      type: string
+    - name: description
+      description: Custom description
+      type: string
+    - name: creator
+      description: Owner, user who requested the import
+      type: string
+  # TODO: baseImage selection with osVersion, osName, pythonVersion
+  workspaces:
+    - name: packages-to-install
+
+  tasks:
```

Comment:
To follow the convention with other pipelines, I think Tasks should be split in separate files under `pipelines/tasks`, and added to `pipelines/tasks/kustomization.yaml`